src/django/api/models.py

```diff
@@ -406,142 +408,241 @@ class FacilityClaim(models.Model):
         max_length=200,
         null=False,
         blank=False,
+        verbose_name='Contact person',
         help_text='The contact person for the facility claim')
     email = models.EmailField(
         null=False,
         blank=False,
+        verbose_name='Email',
         help_text='The contact email for the facility claim')
     phone_number = models.CharField(
         max_length=200,
         null=False,
         blank=False,
+        verbose_name='Phone number',
         help_text='The contact phone number for the facility claim')
     company_name = models.CharField(
         max_length=200,
         null=False,
         blank=True,
+        verbose_name='Company name',
         help_text='The company name for the facility')
     website = models.CharField(
         max_length=200,
         null=False,
         blank=True,
+        verbose_name='Website',
         help_text='The website for the facility')
     facility_description = models.TextField(
         null=False,
         blank=True,
+        verbose_name='Description',
         help_text='A description of the facility')
     verification_method = models.TextField(
         null=False,
         blank=True,
+        verbose_name='Verification method',
```

Comment:
Very trivial but I just read that the Django doc recommend not capitalizing the first letter of these verbose name fields since Django will capitalize them as it needs: https://docs.djangoproject.com/en/2.2/topics/db/models/#verbose-field-names

However, if it doesn't matter feel free to ignore this.

Reply:
I will update these to be consistent with the Django docs and your new fields in #650. I will also make the required update to make sure that we do the capitalization when preparing the email context object.

Reply:
Done in fixups 55bd009, 8932d49, and 227fd9a